server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/ImapKeywordsConsistency.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Impact of IMAP on JMAP keywords consistency
```

Comment:
Could we add apache License in features file ?

(Using # for comments?) Could we add a first commit in this PR to add it in every feature file?